### ISSUE TITLE
Loading Indicator Accessibility Improvements: Live Regions

### ISSUE THREAD

**Issue by GCHQ-Developer-299** [mi6/ic-ui-kit#1694]
## Summary 
The [Loading Indicator Component](https://design.sis.gov.uk/components/loading-indicator) doesn't announce itself to screen readers. Curiously, the "changing label" variant does announce itself.

## 💬 Description 
Proposal:

- [ ] By default, give Loading Indicators `aria-live="polite"` so they announce to screenreaders
- [ ] Add a 'silent' prop that defaults to false, which would conditionally remove the aria live (ensure guidance advises against this) in the rare case that something might need to load silently
- [ ] bonus: deprecate 'name' prop and replace with 'accessibleLabel' to bring Loading Indicator inline with modern components

## 💰 Use value 
When a screenreader user triggers part of a page to load, and a developer has one of our Loading Indicators render, it will announce the label to the user, letting them know that something is loading.

## Additional info 
This issue was raised in an accessibility testing session
